tools/gluten-it/common/src/main/scala/org/apache/gluten/integration/action/Queries.scala

```diff
@@ -141,17 +153,18 @@ object Queries {
   }
 
   private def runQuery(
-      runner: QueryRunner,
-      creator: TableCreator,
-      session: SparkSession,
-      id: String,
-      desc: String,
-      explain: Boolean,
-      randomKillTasks: Boolean): TestResultLine = {
+                        runner: QueryRunner,
+                        creator: TableCreator,
+                        session: SparkSession,
+                        id: String,
+                        desc: String,
+                        explain: Boolean,
+                        metricMapper: MetricMapper,
+                        randomKillTasks: Boolean): TestResultLine = {
```

Comment:
Seems this is not a consistent code style. And I note there are some other alignment changes that may be made by IDE. Can we enable spotless plugin in gluten-it module?

Reply:
Thank you for catching! `gluten-it` is not subject to the main repo's linter configurations, which definitely needs to improve. I'll update this PR and enable code style checks in a future PR.